rollup.config.js

```diff
@@ -1 +1,13 @@
-module.exports = require('skatejs-build/rollup.config');
+const config = require('skatejs-build/rollup.config');
```

Comment:
👍

Do you think it'd be a good idea to ignore peer dependencies here, as well? https://github.com/skatejs/build/blob/master/rollup.config.js#L24